core/src/com/ericc/the/game/map/Map.java

```diff
@@ -136,4 +115,18 @@ public void addRoom(Room room) {
     public HashSet<Room> getRooms() {
         return rooms;
     }
+
+    /**
+     * Registers a tile in a fog of war structure, marks it as seen.
+     */
+    public void registerTile(int x, int y) {
```

Comment:
to jest zła nazwa, bo nie sugeruje w żaden sposób że chodzi o fog of war. Ja gdybym to zobaczył pierwszy raz, to pomyślałbym że to się odpala np. przy wczytywaniu mapy i rejestruje tile w jakimś spisie tile'ów. Proponuję markAsSeenByPlayer